components/AccessWarning/AccessWarning.tsx

```diff
@@ -18,15 +18,17 @@ const warnings: Record<WarningCause, string> = {
   unknown_cause: "Something went wrong.",
 };
 
-const AccessWarning = ({ cause = "unknown_cause" }: AccessWarningProps) => (
-  <div suppressHydrationWarning>
-    <Typography variant="subtitle1">
-      App can&apos;t be accessed outside of the Saleor Dashboard
-    </Typography>
-    <Typography variant="subtitle2" style={{ marginTop: "2rem" }}>
-      ❌ {warnings[cause]}
-    </Typography>
-  </div>
-);
+function AccessWarning({ cause = "unknown_cause" }: AccessWarningProps) {
+  return (
```

Comment:
Why are we using the old `function xyz...` format?

Reply:
Please let's not 🙈

Reply:
@Cloud11PL it's a style guide from the https://github.com/saleor/react-storefront/

Reply:
Are we sure we want it in our template though?

Reply:
Don't have a strong opinion regarding this rule, but would like to have consistency between our projects.